ledger/participant-integration-api/src/main/scala/platform/store/dao/events/ContractStateEvent.scala

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.platform.store.dao.events
+
+import com.daml.ledger.participant.state.v1.Offset
+import com.daml.lf.transaction.GlobalKey
+
+sealed trait ContractStateEvent extends Product with Serializable {
+  def eventOffset: Offset
+  def eventSequentialId: Long
+}
+object ContractStateEvent {
+  final case class Created(
+      contractId: ContractId,
+      contract: Contract,
+      globalKey: Option[GlobalKey],
+      stakeholders: Set[Party],
+      eventOffset: Offset,
```

Comment:
Please add the createLedgerEffectiveTime in here (`ledger_effective_time` in the `participant_events` table)

Reply:
I've added `ledgerEffectiveTime` for brevity but I can change if you prefer the full form.